packages/liveblocks-core/src/client.ts

```diff
@@ -614,6 +615,9 @@ export function createClient<TUserMeta extends BaseUserMeta = BaseUserMeta>(
         cacheStore,
         usersStore,
         roomsInfoStore,
+        get rooms() {
+          return Array.from(roomsById.keys());
+        },
```

Comment:
@nvie I have exposed a getter that returns the list of room ids. Also curious if it is okay to do so?

Reply:
It works, but as a user I think I would prefer getters to be relatively simple "property like" lookups, and not do a computation like this (and return a different reference every time). Would exposing this as a `getRooms()` function not help? I think it better signifies that if you want to read it multiple times, it's better to assign the result to an intermediate variable?

Also… maybe rename this to `getRoomIds()` instead of `getRooms()`?